main.rb

```diff
@@ -0,0 +1,54 @@
+require './app'
+
+def main
+  books = []
+  student = []
+  people = []
+  rental = []
+  option(books, student, people, rental)
+end
+
+def start_up
+  puts 'Please choose an option by entering a number:'
+  puts '1 - List all books'
+  puts '2 - List all people'
+  puts '3 - Create a person'
+  puts '4 - Ceate a new book'
+  puts '5 - Ceate a new rental'
```

Comment:
You have a typo here. You have `Ceate` instead of `Create` for `Ceate a new book` and `Ceate a new rental`.